components/figma/figma.app.mjs

```diff
@@ -99,5 +99,35 @@ export default {
         data,
       }));
     },
+    async deleteComment(fileId, commentId, ctx = this) {
+      return axios(ctx, this._getAxiosParams({
+        method: "DELETE",
+        path: `/v1/files/${fileId}/comments/${commentId}`,
+      }));
+    },
+    async createHook(
+      event_type,
+      team_id,
```

Comment:
Although seems unimportant, I think it would be better to conform to the guidelines here

```suggestion
      eventType,
      teamId,
```
```js
        data: {
          event_type: eventType,
          team_id: teamId,
          endpoint,
          passcode,
        },
```